lib/vagrant-libvirt/action/shutdown_domain.rb

```diff
@@ -46,6 +46,8 @@ def call(env)
 
           return @app.call(env) if env[:result]
 
+          return @app.call(env) if env[:force_halt]
```

Comment:
I'm inclined to think that checking `env[:force_halt]` should be done in the action.rb to call `HaltDomain` and next instead of handling this in ShutdownDomain, see adding the check at https://github.com/vagrant-libvirt/vagrant-libvirt/blob/06fa756b98f3a86254e3bd2cf2b17f2bd5829023/lib/vagrant-libvirt/action.rb#L197 and call `b3.use HaltDomain` followed by `next` instead.